PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+**Note:** Filling out this template is required. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the
```

Comment:
This note might be beneficial for two reasons: 

1. The first is code-related. Any pull requests we receive from the community will be small enough to review without being put on the backlog for too long. If there are too many files changed, we might not review the PR.

2. The second is description related. If not enough information is provided (i.e. sections of the pull request template are not filled out), then the PR can be closed. Whoever submits the PR will need to thoroughly convince the team why it should be included in WorldWind. This is both for the WorldWind team and community's sake. Otherwise, we might spend too time deciding whether or not to merge the pull request, just because the information in the PR was too vague.